src/org/company/MySuperLongClass.java

```diff
@@ -0,0 +1,18 @@
+package org.company;
+
+import org.company.abstracts.AbstractMyClass;
+
+/**
+ * @since Current.Project.Version
+ */
+public class MySuperLongClass {
+
+    public MySuperLongClass() {
+        //
+    }
+
+    public MySuperLongClass(final int one, final String two, final Object three, final long four, final double five, final float six, final MyEnum seven, final MyClass eight, final AbstractMyClass nine) {
```

Comment:
May  we group them in side the context when we hit a maximum line length?

like:
```
function(
var one,   var two
var three, var four
)
```
The lines are grouped in equal manners and aligned into a table grid.